src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -704,6 +704,12 @@
     text: az acr manifest list-metadata -r myregistry -n hello-world
   - name: List the metadata of the manifests in the repository 'hello-world'.
     text: az acr manifest list-metadata myregistry.azurecr.io/hello-world
+  - name: List the metadata of all manifests (tagged/untagged) in the repository 'hello-world'.
+    text: az acr manifest list-metadata -r myregistry -n hello-world --query "[]"
+  - name: List the metadata of untagged manifests in the repository 'hello-world'.
```

Comment:
I think adding only one case should suffice. But if there's a need for more please explain.